src/api/v3/response.ts

```diff
@@ -0,0 +1,63 @@
+import { RecommendationsQuery } from '../../generated/graphql/types';
+import { components, paths } from '../../generated/openapi/types';
+import { Unpack } from '../../types';
+import { appendUtmSource } from '../desktop/recommendations/response';
+
+// unpack GraphQL generated types from RecommendationsQuery
+type GraphRecommendation = Unpack<
+  RecommendationsQuery['newTabSlate']['recommendations']
+>;
+
+// unpack exact OpenAPI generated types for API response
+export type GlobalRecsResponse =
+  paths['/v3/firefox/global-recs']['get']['responses']['200']['content']['application/json'];
+type LegacyFeedItem = components['schemas']['LegacyFeedItem'];
+
+export const mapRecommendation = (
+  recommendation: GraphRecommendation
+): LegacyFeedItem => {
+  const encodedImageUrl = encodeURIComponent(
+    recommendation.corpusItem.imageUrl
+  );
+
+  return {
+    id: recommendation.tileId,
+    url: appendUtmSource(recommendation.corpusItem.url),
+    title: recommendation.corpusItem.title,
+    excerpt: recommendation.corpusItem.excerpt,
+    domain: recommendation.corpusItem.publisher,
+    raw_image_src: recommendation.corpusItem.imageUrl,
+    image_src: `https://img-getpocket.cdn.mozilla.net/direct?url=${encodedImageUrl}&resize=w450`,
```

Comment:
nit: if we don't expect this url to change we can throw it somewhere in a config file too but it's fine this way too.